pkg/client/client.go

```diff
@@ -35,7 +35,7 @@ import (
 	openshiftsecurityclientset "github.com/openshift/client-go/security/clientset/versioned"
 	"github.com/pkg/errors"
 	appsv1 "k8s.io/api/apps/v1beta2"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
why did we include this change? the name of this package is already `v1` by default: https://github.com/kubernetes/api/blob/master/core/v1/doc.go#L21

Reply:
hm, good question, this was reformatted automagically by `goimports` :thinking:

Reply:
ack, even if I remove that qualifier goimports insists to reinsert it